core/src/fr/hammons/slinc/Tools.scala

```diff
@@ -1,58 +1,22 @@
 package fr.hammons.slinc
 
-import java.nio.file.{Files, Paths}
-import types.{os, OS}
 import scala.sys.process.*
+import java.nio.file.Path
 
 object Tools:
-  private val appDataStore =
```

Comment:
some routines are moved to https://github.com/markehammons/slinc/pull/84/files#diff-2f376490fbdd79c80a332b26411aa9972c3adadd5781b684fff3234e36a55f8cR66